modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/BulkResourceManager.java

```diff
@@ -110,6 +110,27 @@ public SCIMResponse listWithGET(UserManager userManager, String filter, int star
         return null;
     }
 
+    /**
+     * Method to list with BulkResourceManager
+     *
+     * @param userManager       User manager
+     * @param filter            Filter to be executed
+     * @param startIndexInt     Starting index value of the filter
+     * @param countInt          Number of required results
+     * @param sortBy            SortBy
+     * @param sortOrder         Sorting order
+     * @param domainName        Domain name
+     * @param attributes        Attributes in the request
+     * @param excludeAttributes Exclude attributes
+     * @return SCIM response
+     */
+    @Override
+    public SCIMResponse listWithGET(UserManager userManager, String filter, Integer startIndexInt, Integer countInt,
```

Comment:
No need to override this with the default impl. as this class already inherits this default implementation.

Reply:
fixed: 976ed9b24712c10efd0ef8d67cae983f30570d0d